core/domain/rights_domain.py

```diff
@@ -330,6 +330,76 @@ class ExplorationRightsChange(change_domain.BaseChange):
     ALLOWED_COMMANDS = feconf.EXPLORATION_RIGHTS_CHANGE_ALLOWED_COMMANDS
 
 
+class CreateNewExplorationRightsCmd(ExplorationRightsChange):
+    """class representing the ExplorationRightsChange's
```

Comment:
Start docstrings with capital letters, here and elsewhere.

Reply:
Done!